lib/ttnt/tasks.rb

```diff
@@ -53,4 +50,6 @@ def install
   end
 end
 
-TTNT::TaskHelper.install_tasks
+# Define tasks when this file is required (this file is intended to be required from Rakefile)
+# ref: https://github.com/bundler/bundler//blob/master/lib/bundler/gem_tasks.rb
+TTNT::TaskHelper.define_tasks
```

Comment:
With the comment you left regarding this line on #6, this is not needed. Actually Bundler defines the `install_tasks` method and call it at the bottom of the file so people just have to require the `bundler/gem_tasks` file and all the tasks are defined.

Here, you are telling in the README that people should handle the definition themselves with `TTNT::TestTask.new(t)`. Currently you are defining only a single task with this line and the `TTNT::TestTask.new(y)` call won't create any new task.

I think this line should be called inside `TTNT::TestTask#initialize`. Then you will have to handle the given name so we can have several tasks ; at the moment, we can only have `ttnt:test` and `ttnt:anchor` but you can handle that later in another pull request!